internal/api_client/task_tracker.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2024 Broadcom. All Rights Reserved.
+// SPDX-License-Identifier: MPL-2.0
+
+package api_client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/vmware/terraform-provider-vcf/internal/constants"
+	"github.com/vmware/vcf-sdk-go/client"
+	"github.com/vmware/vcf-sdk-go/client/tasks"
+	"github.com/vmware/vcf-sdk-go/models"
+	"time"
+)
```

Comment:
This is more idiomatic:

```suggestion
import (
	"context"
	"errors"
	"fmt"
	"time"
	
	"github.com/hashicorp/terraform-plugin-log/tflog"
	"github.com/vmware/vcf-sdk-go/client"
	"github.com/vmware/vcf-sdk-go/client/tasks"
	"github.com/vmware/vcf-sdk-go/models"
	
	"github.com/vmware/terraform-provider-vcf/internal/constants"
)
```

Reply:
Agreed, I'll see if I can find some way to automate import formatting in my IDE.
In any case I'll fix these in the next iteration